it/src/test/java/org/eclipse/jkube/integrationtests/openliberty/OpenLibertyK8sITCase.java

```diff
@@ -86,7 +81,7 @@ void k8sBuild() throws Exception {
   }
 
   @Test
-  @Order(2)
+  @Order(1)
```

Comment:
Is this intentional?

Reply:
what's gonna happen to `k8sBuild()`. Does this mean both of these would execute parallelly?

Reply:
> Is this intentional?

Yes, it is.

> what's gonna happen to k8sBuild(). Does this mean both of these would execute parallelly?

Depends on the configuration for the test runner, but given certain circumstances they will. In case they are not executed in parallel, they will be executed in "random" order.

As we've discussed in other occasions, having tests that require to be executed in a given order is an anti-pattern. In our test suite we're doing it as a trade-off to avoid using CI resources and lower the execution time (in general). Nonetheless, we need to identify those tests that can run in any order.
So in this case, running `k8s:resource` or `k8s:build` randomly or even in parallel, should not affect the result of the test.